### PR TITLE
feat: add env input - #25

### PR DESCRIPTION
**Pull request by prescottprue**
* adds `env` input (closes [cypress-io/actions #25](https://github.com/cypress-io/github-action/issues/25))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,28 @@ jobs:
...
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Cypress run with env
+        uses: cypress-io/github-action@v1
+        with:
+          env: host=api.dev.local,port=4222
+```
+
+For more information, visit [the Cypress command-line docs](https://docs.cypress.io/guides/guides/command-line.html#cypress-run-env-lt-env-gt).
```
Comment (bahmutov) [inline README.md]: just a tip: to keep links working we use `on.cypress.io` redirects, which are usually just title of the page (plus any hashes). So this link would be more stable as https://on.cypress.io/command-line#cypress-run-env-lt-env-gt

File: dist/index.js [hunk 2]
```diff
@@ -1907,6 +1907,14 @@ const runTests = () => {
       cmd.push(browser)
     }
 
+    const envInput = core.getInput('env')
+    if (envInput) {
+      // TODO should env be quoted?
+      // If it is a JSON, it might have spaces
+      cmd.push('--env')
+      cmd.push(envInput)
```
Comment (bahmutov) [inline dist/index.js]: hmm, yes, might need quotes, maybe single quotes, ughh, will require testing across OS

Comment (prescottprue) [reply dist/index.js]: When entering JSON it should already be wrapped in it's own string right?

For now do you think we should add a note to the README about potential issues or did you want to improve the handling/testing before releasing it?

**Comment by bahmutov** [comment]
I’m good with it as is - if users find a problem we will fix it :)

**Pull request merged**